ui/apps/platform/src/hooks/usePaginatedQuery.ts

```diff
@@ -84,8 +84,8 @@ export function usePaginatedQuery<Item, ItemKey>(
     // the `fetchPageHandler` function below relies on the reference of `dedupKeyFn`. By storing these
     // in state, it removes the burden of wrapping in a useCallback or useMemo from the caller.
     const [{ debounceRate, dedupKeyFn, onError, manualFetch = false }] = useState(initialOptions);
-    const [isRefreshingResults, setIsRefreshingResults] = useState(false);
-    const [isFetchingNextPage, setIsFetchingNextPage] = useState(false);
+    const [isRefreshingResults, setIsRefreshingResults] = useState(true);
+    const [isFetchingNextPage, setIsFetchingNextPage] = useState(true);
```

Comment:
This makes sense as a default, but I think the semantics will be weird if `initialOptions.manualFetch` is set to `true`. In that case, the hook will declare that a fetch is in progress before one ever is initialized. Can we default this to `true` in all cases except for when `initialOptions.manualFetch` is set to `true`?